Kevin Williams PA1

```diff
@@ -0,0 +1,365 @@
+# COMP3410 Program Template
+# Author: Kevin Williams
+# Assignment: PA[X]
+# Date: Date of submission
+
```

Comment:
Please don't put everything together into 1 file.  I have to run these to evaluate them.  Normally, if it doesn't compile I'll mark it 0 and move on, but I'll separate the code for you this time.